app/services/auth.server.ts

```diff
@@ -60,8 +61,15 @@ if (process.env.DEV_LOGIN === "true") {
         try {
           const user = await createOrLoginWithDev(secret);
           return user;
-        } catch (e) {
-          console.log(e);
+        } catch (error: unknown) {
+          if (error instanceof Error) {
+            sentryException({
+              message: error.message,
+              extra: {
+                loginMethod: "dev",
```

Comment:
does this need to change for prod?

Reply:
Nop, this is the method the user used to login, not the environment, that's set on set up

Reply:
hmm, but should it not change depending on method?

Reply:
Yeah but this is on the dev method

Reply:
I haven't seen the if in the diff github showed me lol